compiler/testdata/errors.go

```diff
@@ -16,14 +16,6 @@ type Uint uint32
 //go:wasmimport modulename validparam
 func validparam(a int32, b uint64, c float64, d unsafe.Pointer, e Uint)
 
-// ERROR: //go:wasmimport modulename invalidparam: unsupported parameter type int
-// ERROR: //go:wasmimport modulename invalidparam: unsupported parameter type string
-// ERROR: //go:wasmimport modulename invalidparam: unsupported parameter type []byte
-// ERROR: //go:wasmimport modulename invalidparam: unsupported parameter type *int32
-//
-//go:wasmimport modulename invalidparam
-func invalidparam(a int, b string, c []byte, d *int32)
```

Comment:
They're no longer valid with the relaxed types in wasm args.

Reply:
I'd prefer if these tests were updated instead of removed altogether. ABIs are tricky to get right, and I'd like to have some tests for them.
For example: slices, maps, structs, and func values are still not allowed.

Reply:
It would still be nice to update these tests instead of removing them.

Reply:
Agreed. I'll TAL

Reply:
I updated the tests, will submit a PR.

Reply:
https://github.com/dgryski/tinygo/pull/17